tests/chroma/conftest.py

```diff
@@ -1,8 +1,22 @@
 """Configuration for the Mirascope chroma module tests."""
+
+
 import pytest
 
 from mirascope.chroma.types import ChromaSettings
 from mirascope.chroma.vectorstores import ChromaVectorStore
+from mirascope.rag import BaseEmbedder
+
+
+class TestEmbedder(BaseEmbedder):
```

Comment:
can you name it this? I often run into issues naming a class Test* in a test file

Reply:
it didnt complain but i will change it